pddlgym/tests/test_pddlenv.py

```diff
@@ -122,7 +122,41 @@ def test_pddlenv_hierarchical_types():
     print("Test passed.")
 
 
+def test_get_all_possible_transitions():
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    domain_file = os.path.join(
+        dir_path, 'pddl', 'test_probabilistic_domain.pddl')
+    problem_dir = os.path.join(dir_path, 'pddl', 'test_probabilistic_domain')
+
+    env = PDDLEnv(domain_file, problem_dir, raise_error_on_invalid_action=True,
+                  dynamic_action_space=True)
+
+    obs, _ = env.reset()
+    action = env.action_space.all_ground_literals(obs).pop()
+    states = env.get_all_possible_transitions(action)
```

Comment:
Minor: this isn't actually a list of states, right? It's a list of (state, reward, done, debug) tuples, if I understand the code correctly. So maybe change variable name to "transitions" or something, so that we don't confuse others looking at this example in the future :)